### PR TITLE
Fix CSR debug mode not properly initializing when no CSR initials points are set

### PR DESCRIPTION
**Pull request by NFSS10**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-sdk/issues/429 |
| Dependencies | -- |
| Decisions | Fixed issue where CSR debug tools wouldn't initialize if the CSR initials were enabled and the CSR initials points array had no points |

### CODE REVIEW

File: src/js/visual/configurator-csr.js [hunk 1]
```diff
@@ -1025,34 +1025,37 @@ ripe.ConfiguratorCsr.prototype._initDebug = function() {
...
-            const linePoints = curve.getPoints(pointsNum);
-            this.debugRefs.renderedInitials.line = new window.THREE.Line(
-                new window.THREE.BufferGeometry().setFromPoints(linePoints),
-                new window.THREE.LineBasicMaterial({ color: 0x0000ff })
-            );
-
-            this.debugRefs.renderedInitials.group.add(this.debugRefs.renderedInitials.line);
-        }
+        // ensures it has the minimum number of points
+        if (this.initialsRefs.renderedInitials.points > 1) {
```
Comment (NFSS10) [inline src/js/visual/configurator-csr.js:1029]: **For the reviewers:**
The fix is very small, it was just adding this check, the rest of the changes is just the code now being inside this if brackets

Comment (joao-conde) [inline src/js/visual/configurator-csr.js:1029]: Do you mean to do "points.length" ? Or points is a counter?

Comment (NFSS10) [reply src/js/visual/configurator-csr.js:1029]: Ups, made a mistake when moving the fix to this branch 👀

File: src/js/visual/configurator-csr.js [hunk 2]
```diff
@@ -1025,34 +1025,37 @@ ripe.ConfiguratorCsr.prototype._initDebug = function() {
...
-            const linePoints = curve.getPoints(pointsNum);
-            this.debugRefs.renderedInitials.line = new window.THREE.Line(
-                new window.THREE.BufferGeometry().setFromPoints(linePoints),
-                new window.THREE.LineBasicMaterial({ color: 0x0000ff })
-            );
-
-            this.debugRefs.renderedInitials.group.add(this.debugRefs.renderedInitials.line);
-        }
+        // ensures it has the minimum number of points
+        if (this.initialsRefs.renderedInitials.points.length > 1) {
```
Comment (joao-conde) [inline src/js/visual/configurator-csr.js:1029]: Ok, sorry again, but isnt the minimum 1? In other words, dont you want this to be > 0?
If it is some ad-hoc maximum value should that be hardcoded or softcoded as an argument?

Comment (NFSS10) [reply src/js/visual/configurator-csr.js:1029]: No, I told you the min and max can vary.

Comment (NFSS10) [reply src/js/visual/configurator-csr.js:1029]: No, I wan't the minimum to be 2 as you can only have a line with a minimum of 2 points.

**Pull request merged**